src/models/GatewayAccount.class.js

```diff
@@ -1,17 +1,20 @@
-const { GatewayAccountCredential, CREDENTIAL_STATE } = require('@models/gateway-account-credential/GatewayAccountCredential.class')
+const { GatewayAccountCredential } = require('@models/gateway-account-credential/GatewayAccountCredential.class')
+const CREDENTIAL_STATE = require('@models/credential-state')
 const Worldpay3dsFlexCredential = require('@models/gateway-account-credential/Worldpay3dsFlexCredential.class')
+const { InvalidConfigurationError } = require('@root/errors')
+const pendingCredentialStates = [CREDENTIAL_STATE.CREATED, CREDENTIAL_STATE.ENTERED, CREDENTIAL_STATE.VERIFIED]
```

Comment:
Would these be better exported by `@models/credential-state`? or are they only ever going to be used here?

Reply:
given this is just a subset of the constant I didn't think it was necessary to export it specifically

Reply:
sure, can always be moved if it ends up being needed somewhere else as well